docs/content/authentication.md

```diff
@@ -60,6 +60,37 @@ grr config set synthetic-monitoring.logs-id # Metrics instance ID
 ```
 Your stack ID is the number at the end of the url when you view your Grafana instance details, ie. `grafana.com/orgs/myorg/stacks/123456` would be `123456`. Your metrics and logs ID's are the `User` when you view your Prometheus or Loki instance details in Grafana Cloud.
 
+## Configuring Targets
```

Comment:
Nit: Weird that the file is called `authentication.md`

Reply:
@julienduchesne I've moved auth->setup/config, and put in a redirect page for any old links.